bench.py

```diff
@@ -138,11 +157,10 @@ def parse_args(parser, size=None, loop_types=(),
                         action='store_true',
                         help='Check finiteness in sklearn input check'
                              '(disabled by default)')
-    parser.add_argument('--output-format', type=str, default='csv',
-                        choices=('csv', 'json'),
-                        help='Output format: csv (default) or json')
-    parser.add_argument('--time-method', type=str, default='mean_min',
-                        choices=('box_filter', 'mean_min'),
+    parser.add_argument('--output-format', type=str, default='json',
+                        choices=('json'), help='Output format: json')
```

Comment:
Minor: csv was nice for short output, is it hard to save it?

Reply:
It doesn't work everywhere, it's hard to support for new benchmarks, I like the current output. In the future, we can add console output, but we need to think about how to do it better